.env

```
@@ -0,0 +1,3 @@
+TODO_PORT = 7540

```

В энв файле лучше не ставить пробелы
```suggestion
TODO_PORT=7540
```

---

Исправил